src/AkismetField.php

```diff
@@ -184,10 +192,17 @@ public function getIsSpam()
         // Check result
         /** @var AkismetService $api */
         $api = AkismetSpamProtector::singleton()->getService();
-        $this->isSpam = $api && $api->isSpam($content, $author, $email, $url);
+        $this->isSpam = false;
+        try {
+            $this->isSpam = $api && $api->isSpam($content, $author, $email, $url);
+        } catch (Exception $e) {
+            //if the network response fails, it still needs to be true/false
+            $this->isSpam = (bool) $this->config()->is_spam_when_response_fails;
```

Comment:
Do we want to do something with the exception, even if it's logging?

Reply:
To be honest, I'm not too concerned with logging the error, I have other tools to capture that information. The current module doesn't handle exceptions at all, and I'd prefer to keep that as a separate feature if it's required.

This is causing production issues for several of our CWP clients and I'd appreciate a review and merge to fix this supported module. Thanks.

Reply:
Can we just add a `Injector::inst()->get(LoggerInterface::class)->error($e->getMessage());` rather than just swallowing it? If you're marking authentic comments as spam, I think you want some awareness of it.

Reply:
The exception message only says "invalid" when caught. Is there anything else I should include with the message? I'm hesitant to include the actual values of the four checked fields, as it means logging some level of personal user information..

Reply:
I'll append a string from the en.yml file containing a more specific reason.

Reply:
If that's the case, then let's just provide some context in the error.

```
Injector::inst()->get(LoggerInterface::class)->error(sprintf(
  'Akismet marked content as spam due to network error. Response from service: %s',
  $e->getMessage()
)
```

No need to i18n an error message.

Reply:
There's an existing i18n I've used instead. Squashed and pushed